RocketReserver/LaunchListView.swift

```diff
@@ -7,9 +7,9 @@ struct LaunchListView: View {
     NavigationView {
       List(viewModel.launches) { launch in
         LaunchRow(launch: launch)
-          .task(id: viewModel.loadState) {
+          .onAppear {
             guard launch.id == viewModel.launches.last?.id else { return }
-            await viewModel.loadNextPage()
+            Task { await viewModel.loadNextPage() }
```

Comment:
I'm changing this to an `onAppear` block specifically because I want to make sure that the widely-used `onAppear` pattern (which can trigger at the wrong times + multiple times) is valid and will continue to operate as expected. 

What I've noticed is _yes_, this will work without needing to also `onChange` the `loadState`.